data-products/_docs/product-governance.md

```diff
@@ -0,0 +1,109 @@
+
+- `date-uploaded` - the retention period is based on the date & time the data was added to the platform; or
+- `data-date-key` - the retention period is based on dates within the data itself. For this strategy the [data dictionary](data-dictionary.md) must contain key date fields for all datasets.
+
+For these strategies, you must also supply a time unit and values. Supported values for `retention-unit` are:
+
+- `months`
+- `years`
+
+The `retention-value` should be a whole number indicating how many of the `retention-unit` elapses before the retention policy is applied. For example:
+
+```yaml
+retention:
+  data-platform.justice.gov.uk/retention-strategy: "data-date-key"
+  data-platform.justice.gov.uk/retention-unit: "years"
+  data-platform.justice.gov.uk/retention-value: 3
+```
+
+Would be implemented as _"delete any data older than 3 years, based on the date field for each table"_.
+
+You can also specify `forever` for data which is allowed to be kept indefinitely - for example anonymised management information or published statistics. If `forever` is specified, the `retention-unit` and `retention-value` values are ignored - they are included above to provide a more complete example.
+
+Data products owners will be notified of retention policies being applied using the contact information supplied in the [data product specification](product-specification.md).
+
+### Data updates
+
+You should indicate to the platform, and to users, how frequently the data will be updated using the `update-frequency` attribute. This requires two values:
+
+- `data-platform.justice.gov.uk/update-unit`
+- `data-platform.justice.gov.uk/update-value`
+
+Supported values for `update-unit` are:
+
+- `days`
+- `weeks`
+- `months`
+
+The `update-value` should be a whole number indicating how many of the `update-unit` will elapse before the next upate is sent. For example:
+
+```yaml
+update-frequency:
+  data-platform.justice.gov.uk/update-unit: "months"
+  data-platform.justice.gov.uk/update-value: 3
+```
+
+Would be implemented as _"new data will be supplied every three months"_. We don't support multiples of years, but a dataset with annual updates can be specified as having a 12 month update cycle.
+
+If a data update cycle is missed, data product owners will be notified via the contact information supplied in the [data product specification](product-specification.md).
+
+### Data lineage
+
+It is important to users of the data that they know where it has come from. This is the purpose of the `lineage` collection of attributes. These attributes are generally free text, so you are free to interpret their usage in the way you think is most helpful to users.
+
+- `data-platform.justice.gov.uk/top-level-domain` defines at a high level which part of the organisation the data has come from. For example "HMCTS", "HMPPS", "LAA", "OPG", "HQ"
```

Comment:
Is Top Level Domain too technical? Maybe people refer to these as "organisations" or a different term?